src/base/torrentfileguard.h

```diff
@@ -62,7 +63,7 @@ class TorrentFileGuard: private FileGuard
     void markAsAddedToSession();
     using FileGuard::setAutoRemove;
 
-    enum AutoDeleteMode     // do not change these names: they are stored in config file
+    enum AutoDeleteMode: int     // do not change these names: they are stored in config file
```

Comment:
No, it is implementation defined smallest possible integral type, that can fit all the values. But it is `int` by default for a scoped enumeration.